src/assets/js/protocol/protocol-utils.js

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+// create namespace
+if (typeof Mozilla === 'undefined') {
+    var Mozilla = {};
+}
+
+(function() {
+    'use strict';
+
+    var Utils = {};
+
+    // matches() polyfill
+    // TODO: where should this go?
+    if (!Element.prototype.matches) {
```

Comment:
Is there a better place to put a polyfill?

Reply:
Hmm, we don't yet use any polyfills in Protocol, so we don't have a place to store them. We should also document what polyfills we're using which will be really important (maybe something under documentation in the menu?).

This particular piece of code doesn't seem to be a polyfill though, it's more smushing vendor prefixes into a single reference. There does seem to be a [polyfill listed on MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/matches#Polyfill) though?

Reply:
Maybe it's more of a vendorfill? Is that a thing?

Reply:
Leaving this here for now in the interests of getting the footer into use.